brigade.js

```diff
@@ -79,161 +211,52 @@ class Notification {
 
   // Send a new notification, and return a Promise<result>.
   run() {
-    this.count++
-    var j = new Job(`${this.name}-${this.count}`, "deis/brigade-github-check-run:latest");
-    j.imageForcePull = true;
-    j.env = {
-      CHECK_CONCLUSION: this.conclusion,
-      CHECK_NAME: this.name,
-      CHECK_TITLE: this.title,
-      CHECK_PAYLOAD: this.payload,
-      CHECK_SUMMARY: this.summary,
-      CHECK_TEXT: this.text,
-      CHECK_DETAILS_URL: this.detailsURL,
-      CHECK_EXTERNAL_ID: this.externalID
-    }
-    return j.run();
+    this.count++;
+    // Here we are using the mutable 'edge' version of this utility
+    // as an exercise of vetting the current master version of the code in this repo.
+    // It is recommended that immutable tags be used in other cases,
+    // e.g., a proper semver tag of 'vX.X.X' or the short git sha of a particular commit.
+    var job = new Job(`${ this.name }-notification-${ this.count }`, "brigadecore/brigade-github-check-run:edge");
```

Comment:
Fwiw, I've been in the habit of specifying `latest` (latest tagged release) for all non-`brigadecore/brigade-github-app` repos (in other words, only really using edge for the repo that produces the artifact).  Another option, of course, is the most recent immutable semver tag, `v0.1.0`.

Reply:
Nice catch @vdice. Clearly, you can tell exactly _where_ that was copied/pasted from. Fixing this!

Reply:
Fixed.